PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -249,6 +268,27 @@ struct HfTaskFlowCharmHadrons {
     return cent;
   }
 
+  /// Check if the collision is selected
+  /// \param collision is the collision with the Q vector information
+  /// \param bc is the bunch crossing with timestamp information
+  /// \return true if the collision is selected, false otherwise
+  template <o2::hf_centrality::CentralityEstimator centEstimator>
+  bool isCollSelected(CollsWithQvecs::iterator const& collision,
+                      aod::BCsWithTimestamps const&)
+  {
+    float centrality{-1.f};
+    const auto rejectionMask = hfEvSel.getHfCollisionRejectionMask<true, centEstimator, aod::BCsWithTimestamps>(collision, centrality, ccdb, registry);
+
+    /// monitor the satisfied event selections
+    hfEvSel.fillHistograms(collision, rejectionMask, centrality);
+
+    if (rejectionMask != 0) {
+      /// at least one event selection not satisfied --> reject the candidate
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    return rejectionMask == 0;
```

Reply:
This is not superclear to me. If I always return 0, then I always return `false`, which means I accept all the collisions, right? Or am I missing something?

Reply:
I see, thanks for the explanation!